app/views/search/_sitetour_modal.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="modal fade sitetour" id="sitetourModal" style="font-family: FontAwesome;" tabindex="-1" role="dialog">
```

Comment:
I'll add that to the CSS and make it so - we should have a sideline chat about inline CSS at some point.  I want to make sure I know the team rules; I tend to use inline for what appears to be 'very minor' bumps, but that gets subjective.